### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/frontmatter/project-home.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#924] [Announcements]
# https://theme-hope.vuejs.press/zh/config/frontmatter/project-home.html

home 必须设置为 true 以使用项目首页。 title 类型: string 必填: 否 设置页面标题，会用于路径导航、页面增强等。 heroText 类型: string | false 默认值: 

https://theme-hope.vuejs.press/zh/config/frontmatter/project-home.html

**Comment by gydchenxiao** [comment]
主页heroText和二级标题tagline默认字体是白色，在哪里可以修改字体的颜色呢，谢谢hope大佬

**Comment by yuanbenxin** [comment]
这里链接怎么又说不支持相对路径了？？？
https://theme-hope.vuejs.press/zh/cookbook/vuepress/markdown.html#%E9%93%BE%E6%8E%A5不是说要尽可能用相对路径吗？
另：绝对路径跟相对路径怎么用的

**Reply by Mister-Hope** [reply]
所有在 frontmatter 以及其他在数据结构中设置的的媒体资源都只能用绝对路径或者完整路径。打包器不会理解数据的特定字段是一个需要被打包的资源，所以显然就不会工作。

**Comment by yuanbenxin** [comment]
还是麻烦问一下，绝对路径可以给个示例吗，这里的文件路径我没有理解，绝对路径是要从src开始直到目标文件，像“/src/…/*.*”？还是从磁盘盘符出发像这样“X:/…/[项目文件夹]/src/…/*.*”？，再次感谢您的回答。
发件人: Mister-Hope ***@***.***>
发送时间: 2025年7月23日 14:57
收件人: vuepress-theme-hope/giscus-discussions ***@***.***>
抄送: 袁本新 ***@***.***>; Comment ***@***.***>
主题: Re: [vuepress-theme-hope/giscus-discussions] https://theme-hope.vuejs.press/zh/config/frontmatter/project-home.html (Discussion #924)


所有在 frontmatter 的媒体资源都只能用绝对路径。打包器不会理解数据的特定字段是一个需要被打包的资源，所以显然就不会工作。
